pkg/authenticator/config/config_test.go

```diff
@@ -0,0 +1,80 @@
+package config
+
+import (
+	"fmt"
+	"github.com/cyberark/conjur-authn-k8s-client/pkg/log"
+	"os"
+	"testing"
+
+	. "github.com/smartystreets/goconvey/convey"
+)
+
+var mockSuccessfulOptions = Options{
+	successfulMockReadSSLCertFunc,
+	 successfulMockNewUserNameFunc,
+}
+
+var environmentValues = map[string]string{
+	"CONJUR_SSL_CERTIFICATE": "testSSLCert",
+	"CONJUR_CERT_FILE": "testSSLCertFile.txt",
+	"CONJUR_AUTHN_URL": "filepath",
+	"CONJUR_ACCOUNT": "testAccount",
+	"CONJUR_AUTHN_LOGIN": "host",
+	"MY_POD_NAMESPACE": "testNameSpace",
+	"MY_POD_NAME": "testPodName",
+	"CONJUR_VERSION": "",
+}
+
+func TestAuthenticator(t *testing.T) {
+	// Set default environment variables
+	for key, value := range environmentValues {
+		err := os.Setenv(key, value)
+		if err != nil {
+			panic(err)
+		}
+	}
```

Comment:
This should have a `defer`ed unsetting of those variables to not pollute other tests